db/migrate/20170822072811_insert_existing_user_ip_address_and_remove_column_from_users.rb

```diff
@@ -0,0 +1,13 @@
+class InsertExistingUserIpAddressAndRemoveColumnFromUsers < ActiveRecord::Migration
+  disable_ddl_transaction!
+  def change
+    User.find_each do |user|
+      all_users_notifications = []
+      user.ip_addresses.each do |ip|
+        all_users_notifications << {ip_address: ip, user: user}
+      end
+      UserIpaddress.create(all_users_notifications)
```

Comment:
Looks like you have the old capitalization here

Reply:
Ahh good catch